src/tree/SubscriptionTreeItem.ts

```diff
@@ -41,7 +41,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         }
 
         if (this.rgsItem.length === 0) {
-            this.rgsItem.push(...(await applicationResourceProviders[0]?.provideResources(this.subscription) ?? []));
+            this.rgsItem.push(...(await applicationResourceProviders['vscode-azureresourcegroups.azureResourceProvider']?.provideResources(this.subscription) ?? []));
```

Comment:
Good question-- we could make it a constant (since we know we'll always need to get rgItems from Resource Groups) or push it into a list of providers/manifests and iterate through the ids?